### PR TITLE
data: mahfuztbt

### PR DESCRIPTION
**Pull request by mahfuztbt**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4929"></a>

### CODE REVIEW

File: data/mahfuztbt.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Mahfuz Ronnie",
+  "type": "personal",
+  "bio": "Aspiring Data Scientist",
+  "avater": "https://github.com/mahfuztbt.png",
```
Comment (priyankarpal) [inline data/mahfuztbt.json:5]: Currently, you don't need to add an `avatar` code. Please delete this line. 
```suggestion

```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by mahfuztbt — COMMENTED** [review]
delete avatar

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

**Comment by mahfuztbt** [comment]
is that okay?

**Comment by priyankarpal** [comment]
> is that okay?

It's ok. Wait for the merge.

**Review by stevenovak123 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**